tests/testsuite/profile_targets.rs

```diff
@@ -596,12 +596,12 @@ fn profile_selection_check_all_targets_test() {
     //
     p.cargo("check --all-targets --profile=test -vv").with_stderr_unordered("\
 [COMPILING] bar [..]
-[RUNNING] `[..] rustc --crate-name bar bar/src/lib.rs [..]--crate-type lib --emit=[..]link[..]-C codegen-units=5 -C debuginfo=2 [..]
+[RUNNING] `[..] rustc --crate-name bar bar/src/lib.rs [..]--crate-type lib --emit=[..]link[..]-C codegen-units=5 [..]
```

Comment:
Would you like an additional test that doesn't use elision for the args and asserts most of the command line, so that it would fail with the old output ?

Reply:
I don't expect to assert the whole command line arguments, as it might be unstable, and we may end up update them frequently. If you've figured out how to do it elegantly, please share!

Reply:
I don't know how elegant it would be, but we could also add assertions that one of the  `-C debuginfo=2` commands is not present (`with_stderr_does_not_contain`) in the output, so that only the new defaults pass both presence and absence assertions ?

Reply:
That is absolutely a not perfect but feasible approach!

Reply:
I've pushed a commit with some of these assertions, let me know what you think, if we want to add these everywhere or not.

Reply:
Thanks. That's good enough!